<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MERemotePartner</class>
 <widget class="QDialog" name="MERemotePartner">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>544</width>
    <height>344</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Manage partners</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>19</x>
     <y>20</y>
     <width>501</width>
     <height>301</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QScrollArea" name="partnersArea">
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="partnersAreaWidget">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>495</width>
           <height>262</height>
          </rect>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="cancelBtn">
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="addPartnersBtn">
        <property name="text">
         <string>Add partners</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="addHostsBtn">
        <property name="text">
         <string>Add hosts</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="disconnectBtn">
        <property name="text">
         <string>Disconnect</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
